subsys/net/lib/sockets/sockets_tls.c

```diff
@@ -56,6 +62,30 @@ LOG_MODULE_REGISTER(net_sock_tls, CONFIG_NET_SOCKETS_LOG_LEVEL);
 #include <zephyr_mbedtls_priv.h>
 #endif
 
+#if defined(CONFIG_WOLFSSL)
+#include <user_settings.h>
```

Comment:
Typically this is included conditionally based on the WOLFSSL_USER_SETTINGS macro and gets included from `wolfssl/wolfcrypt/settings.h`. In the zephyr case I guess it can be forced, but I suspect `WOLFSSL_USER_SETTINGS` is set and is included twice. Also we now assure ssl.h includes settings.h.

Reply:
It seems for my default builds it is not included. I have added a flag guard, if you think that is not sufficient or incorrect let me know and I can take a second crack at it.